Source/WebCore/page/ImageAnalysisQueue.h

```diff
@@ -51,6 +55,10 @@ class ImageAnalysisQueue {
 
     void enqueueIfNeeded(HTMLImageElement&);
 
+    WEBCORE_EXPORT void setImageQueueEmptyHysteresis(std::unique_ptr<PAL::HysteresisActivity>&&);
+
+    bool analysisOfAllImagesOnPageHasStarted() { return m_analysisOfAllImagesOnPageHasStarted; }
```

Comment:
Should be const:
`bool analysisOfAllImagesOnPageHasStarted() const { return m_analysisOfAllImagesOnPageHasStarted; }`